### ISSUE TITLE
[BUG] efb-wizard failed to create the configuration

### ISSUE THREAD

**Issue by jemyzhang** [blueset/ehForwarderBot#173]
efb-wizard failed to create the configurations 
Error log:
```
# efb-wizard
Checking Python version... OK
Checking master channels... OK
Checking slave channels... OK

Welcome to EH Forwarder Bot Setup Wizard.  This program will guide you
to finish up the last few steps to get EFB ready to use.

To use this wizard in another supported language, please change your
system language or modify the language environment variable and
restart the wizard.

Profile: [default] 

Traceback (most recent call last):
  File "/usr/bin/efb-wizard", line 10, in <module>
    sys.exit(main())
  File "/usr/lib/python3.7/site-packages/ehforwarderbot/wizard.py", line 637, in main
    choose_master_channel(data)
  File "/usr/lib/python3.7/site-packages/ehforwarderbot/wizard.py", line 453, in choose_master_channel
    default_config = data.config['master_channel'].split("#")
AttributeError: 'NoneType' object has no attribute 'split'
```

**To Reproduce**  
1. run efb-wizard in terminal

**Expected behavior**  
wizard should be launched and customized configurations should be created

**Version information (please complete the following information):**  
- OS:  alpine:latest
- Detailed Python, EFB and module versions: (you can obtain this by running `ehforwarderbot --version`)

```
EH Forwarder Bot
Version: ehforwarderbot-2.0.0b19.dist-info
Python version:
3.7

Master channel:
 efb_telegram_master-2.0.0b25.dist-info

Slave channel:
efb_wechat_slave-2.0.0a28.dist-info

Middleware:
-
```

**Comment by blueset** [comment]
Attempted to fix this issue in cf3e716. Will release the fix in next version.

**Comment by jemyzhang** [comment]
Thanks for your help! @blueset
